src/diff/diffCommand.ts

```diff
@@ -192,12 +192,20 @@ Exit statuses:
 
   async run(): Promise<void> {
     const { args, flags } = this.parse(DiffCommand);
+    const baseApis = args.base;
+    const newApis = args.new;
+    if (!fs.pathExistsSync(baseApis)) {
+      this.error(`File or directory not found: ${baseApis}`);
+    }
+    if (!fs.pathExistsSync(newApis)) {
+      this.error(`File or directory not found: ${newApis}`);
+    }
```

Comment:
```suggestion
    if (!await fs.pathExists(baseApis)) {
      this.error(`File or directory not found: ${baseApis}`);
    }
    if (!await fs.pathExists(newApis)) {
      this.error(`File or directory not found: ${newApis}`);
    }
```
Using sync methods when you're already in an async function always seems silly to me.